lib/node_modules/@stdlib/stats/base/dists/arcsine/cdf/README.md

```diff
@@ -137,6 +137,99 @@ for ( i = 0; i < 25; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/arcsine/cdf.h"
+```
+
+#### stdlib_base_dists_arcsine_cdf( x, a, b )
+
+Evaluates the cumulative distribution function (CDF) for an arcsine distribution.
+
+```c
+double out = stdlib_base_dists_arcsine_cdf( 0.5, 0.0, 2.0 );
+// returns ~0.333
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value
+-   **a**: `[in] double` minimum support
+-   **b**: `[in] double` maximum support
+
+```c
+double stdlib_base_dists_arcsine_cdf( const double x, const double a, const double b );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/stats/base/dists/arcsine/cdf.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+int main( void ) {
+    double x;
+    double a;
+    double b;
+    double y;
+    int i;
+    
+    for ( i = 0; i < 25; i++ ) {
+        x = ( (double)rand() / (double)RAND_MAX ) * 20.0 - 10.0;
+        a = ( (double)rand() / (double)RAND_MAX ) * 20.0 - 20.0;
+        b = a + ( (double)rand() / (double)RAND_MAX ) * 40.0;
+        y = stdlib_base_dists_arcsine_cdf( x, a, b );
+        printf( "x: %lf, a: %lf, b: %lf, F(x;a,b): %lf\n", x, a, b, y );
+    }
+}
```

Comment:
```suggestion

static double random_uniform( const double min, const double max ) {
    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
    return min + ( v*(max-min) );
}

int main( void ) {
    double x;
    double a;
    double b;
    double y;
    int i;
    
    for ( i = 0; i < 25; i++ ) {
        x = random_uniform( -10.0, 10.0 );
        a = random_uniform( -20.0, 0.0 );
        b = random_uniform( a, a+40.0 );
        y = stdlib_base_dists_arcsine_cdf( x, a, b );
        printf( "x: %lf, a: %lf, b: %lf, F(x;a,b): %lf\n", x, a, b, y );
    }
}
```

The `rand()` noise makes it harder to grok. Especially for dists such as hypergeometric, having a nice fcn abstraction will improve readability, IMO.

Reply:
For the discrete uniform use case, future contributors to other dists can simply round the results. Not perfect, but likely good enough (?!).

Obviously, would be better if we had C implementations of our dists PRNGs.

Reply:
At least if it's a function, we should be able to more readily find-and-replace in the future with something which is better practice.